README.md

```diff
@@ -26,56 +26,67 @@ compromise and trains you in mitigating against these vulnerabilities.
 
 ## Usage
 
+Ensure the [AWS requirements](#aws-configuration) are met and configured.
+
 The quickest way to get up and running is to simply:
 
 <pre>
 source <(curl https://raw.githubusercontent.com/kubernetes-simulator/simulator/master/kubesim)
+sh -x ./kubesim
```

Comment:
the one liner above downloads and sources in one shot causing it to run.  I actually dont think this should be the recommended first run experience fwiw but your `sh` command will bail because it never hits the disk

Reply:
also it's taking kubesim from the tip of master not the latest release.  I don't think that's the right behaviour either.  We run the risk of breaking every newcomer if we push a broken commit to kubesim